Breadcrumbs.py

```diff
@@ -108,4 +141,223 @@ def get_points_by_row(row):
+  sublime.status_message('Breadcrumbs copied to clipboard')
+
+
+if viewevents_available is not True:
+  class BreadcrumbsEventListenerST2(sublime_plugin.EventListener):
+
+    def on_selection_modified(self, view):
+      if get_statusbar_enabled(view.settings()):
+        current_row = view.rowcol(view.sel()[0].b)[0]
+        breadcrumbs = make_breadcrumbs(view, current_row, shorten=True)
+
+        if len(breadcrumbs) > 0:
+          view.set_status('breadcrumbs', get_separator(view).join(breadcrumbs))
+      else:
+        view.erase_status('breadcrumbs')
+
+
+class BreadcrumbsEventListenerST3(sublime_plugin.ViewEventListener):
+
+  @classmethod
+  def is_applicable(cls, settings):
+    return get_statusbar_enabled(settings) and viewevents_available
+
+  def __init__(self, view):
+    self.view = view
+
+    def clear():
+      if get_statusbar_enabled(view.settings()) is not True:
+        view.erase_status('breadcrumbs')
+
+    defaults = sublime.load_settings('Breadcrumbs.sublime-settings')
+    defaults.add_on_change('show_breadcrumbs_in_statusbar', clear)
+    view.settings().add_on_change('show_breadcrumbs_in_statusbar', clear)
+
+  def on_selection_modified(self):
+    current_row = self.view.rowcol(self.view.sel()[0].b)[0]
+    breadcrumbs = make_breadcrumbs(self.view, current_row, shorten=True)
+    self.view.set_status('breadcrumbs', get_separator(self.view).join(breadcrumbs))
+
+
+class BreadcrumbsPopupCommand(sublime_plugin.TextCommand):
+
+  def run(self, edit):
+
+    '''
+    Built to look and work like the ShowScopeName command's popup
+    See show_scope_name.py in the Default package
+    '''
+
+    stylesheet = '''
+      p {
+        margin-top: 0;
+      }
+      a {
+        font-size: 1.05rem;
+      }
+    '''
+
+    template = '''
+      <body id=show-scope>
+          <style>{stylesheet}</style>
+          <p>{breadcrumbs}</p>
+          <a href="copy">Copy</a>
+      </body>
+    '''
+
+    view = self.view
+    current_row = view.rowcol(view.sel()[0].b)[0]
+    breadcrumbs = make_breadcrumbs(view, current_row)
+    escaped_crumbs = []
+    if len(breadcrumbs) > 0:
+      for crumb in breadcrumbs:
+        escaped_crumbs.append(html.escape(crumb, quote=False))
+      breadcrumbs_element = '<br>'.join(escaped_crumbs)
+    else:
+      breadcrumbs_element = '<em>None</em>'
+
+    breadcrumbs_string = get_separator(view).join(breadcrumbs)
+    body = template.format(
+        breadcrumbs=breadcrumbs_element,
+        stylesheet=stylesheet
+    )
+    view.show_popup(
+        body,
+        max_width=512,
+        on_navigate=lambda x: [
+            copy(view, breadcrumbs_string),
+            view.hide_popup()
+        ]
+    )
+
+  def is_visible(self):
+    return minihtml_available
+
+
+class BreadcrumbsPhantomCommand(sublime_plugin.TextCommand):
+
+  def __init__(self, view):
+    self.phantoms_visible = False
+    self.view = view
+    self.phantom_set = sublime.PhantomSet(view, 'breadcrumbs')
```

Comment:
This line causes a problem in ST2:
```
Traceback (most recent call last):
  File ".\sublime_plugin.py", line 115, in create_text_commands
  File ".\Breadcrumbs.py", line 249, in __init__
    self.phantom_set = sublime.PhantomSet(view, 'breadcrumbs')
AttributeError: 'module' object has no attribute 'PhantomSet'
```
It happens as soon as I put a cursor anywhere inside the buffer, and not sooner, and only once for each tab that I click on. Probably the ST2 tries to instantiate all text commands for each view, and so this fails, because there sis no such thing like PhantomSet in the old sublime.

The end result is somewhat strange, because the "Show Breadcrumbs" option is visible in the context menu, but is greyed out.

Could you install ST2 and make sure your code works in it?